TribalWars/src/ApplicationCore/Response/Response.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApplicationCore.Response
+{
+    public class Response
+    {
+        public string ResponseMessage { get; set; }
```

Comment:
Or simply "Message"
Response.Messsage will be as much readable as Response.ResponseMessage, but shorter